### PR TITLE
No messages ui

### LINKED ISSUE #964: Need 'no messages' UI for when a grower doesn't have any messages yet

### PR DESCRIPTION
**Pull request by EngrTaofeek**
Adds screen display to show users when the messages are empty. Fixes #964 
Screenshot_20221031_194656

### CODE REVIEW

File: app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt [hunk 1]
```diff
@@ -125,5 +126,8 @@ fun IndividualMessageListScreen(
                 }
             }
         }
+        if (state.messages.isEmpty()) {
```
Comment (Elforama) [inline app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt:129]: This looks good overall. Let's make sure we don't run the list code when it's empty.

**Review by Elforama — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt (hunk 1 above)
Comment (EngrTaofeek) [reply app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt:129]: > This looks good overall. Let's make sure we don't run the list code when it's empty.

Sorry I don't get this

Comment (Elforama) [reply app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt:129]: Let's not run the code to render a list of it's empty

Comment (Elforama) [reply app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt:129]: If list empty show empty state, else show list

Comment (EngrTaofeek) [reply app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt]: Oh I get you now.

**Review by Elforama — APPROVED** [review]
Looks great, thanks for the fix

**Pull request merged**